blazor/heatmap-chart/events.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: Events in Blazor HeatMap Component | Syncfusion
+description: Checkout and learn here all about events in Syncfusion Blazor HeatMap component and much more details.
+platform: Blazor
+control: HeatMap Chart
+documentation: ug
+---
+
+# Events in Blazor HeatMap chart Component
+ 
+This section explains the list of events that will be triggered for appropriate actions in HeatMap. The events should be declared in the HeatMap component using the [HeatMapEvents](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.HeatMap.HeatMapEvents.html).
+
+## CellRendering
+
+The [CellRendering](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.HeatMap.HeatMapEvents.html#Syncfusion_Blazor_HeatMap_HeatMapEvents_CellRendering) event will be triggered before each HeatMap cell is rendered. To know more about arguments of this event, refer [here](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.HeatMap.HeatMapCellRenderEventArgs.html).
+
+The following example demonstrates how to customize the cell value, color and border color of the Heatmap component using the `CellRendering` event.
```

Comment:
The following example demonstrates how to customize the data label, color and border color of the cells using the `CellRendering` event.